seqr/views/utils/orm_to_json_utils.py

```diff
@@ -104,6 +107,7 @@ def _get_json_for_user(user):
         _to_camel_case(field): getattr(user, field) for field in [
         'username', 'email', 'first_name', 'last_name', 'last_login', 'is_staff', 'is_active', 'date_joined', 'id',
     ]}
+    user_json['isAnvil'] = is_google_authenticated(user) if is_anvil else False
```

Comment:
This doesn't fix the issue I pointed out in my last review, it just changed the name of the parameter without fixing the behavior. When you are getting the json for the list of anvil-only collaborators, you want `isAnvil` to be true, regardless of whether or not that user ever has logged in with google. For the current user, you want to make `is_anvil` dependent on how they logged in, but for the collaborator json this should be unconditionally true. Here you should just do `user_json['isAnvil'] = is_anvil`, and then when you call the function for the current user you can do `_get_json_for_user(user, is_anvil=is_google_authenticated(user))`

Reply:
I see the issue you are talking about now. But, in my opinion, not all the collaborators fetched from AnVIL have logged-in with AnVIL yet. So, it still needs to call `is_google_authenticated(collaborator)` to check the `is_anvil` for the collaborator in the list. Do you agree? 

Another reason why I used `check_anvil` as an optional parameter for `_get_json_for_user()` was that I didn't like to change the other places where call `_get_json_for_user()`.

I could change the function to `_get_json_for_user(user, is_anvil=None)` and it does `user_json['isAnvil] =  is_anvil if is_anvil != None else is_google_authenticated(user)`. Which means it will use `is_anvil` if provided. How do you think about it?

Reply:
so the `isAnvil` attribute for collaborators is for how they are displayed on the project page. So if a collaborator has access through Anvil but has never logged in to regular seqr at all, it would be really bad to show them as a local collaborator, given that their access comes purely from anvil. The big difference between the local and anvil sections is that local users can be edited and anvil users can't. So if a user has access through anvil, it doesn't matter how they are logging in at this point, theres no way to remove them from the project in seqr and therefore they should show up in the uneditable anvil section

If you want to leave `_get_json_for_user` (which I totally get) you could go back to using the `check_anvil` defaulted to True the way you did before and leave this line the same where it checks the google authentication if that is True. Then in the function to get  json for collaborators you set `check_anvil` to False, and then when you go through the list of anvil collaborator emails you update the `isAnvil` attribute directly

Reply:
Oh sorry I commented before seeing your latest code change. I think that approach is fine, so feel free to ignore that suggestion